lib/node_modules/@stdlib/stats/base/dists/levy/ctor/benchmark/benchmark.js

```diff
@@ -32,15 +33,22 @@ var Levy = require( './../lib' );
 
 bench( pkg+'::instantiation', function benchmark( b ) {
 	var dist;
+	var len;
 	var mu;
 	var c;
 	var i;
 
+	len = 100;
+	mu = new Float64Array( len );
+	c = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		mu[ i ] = ( uniform( -50.0, 50.0 ) );
```

Comment:
```suggestion
		mu[ i ] = uniform( EPS, 10.0 );
```

No need of outside brackets.